CMakeLists.txt

```diff
@@ -57,6 +57,9 @@ set(CPP_PCP_CLIENT_FLAGS "-Wno-deprecated-declarations -Wno-reorder -Wno-sign-co
 set(CMAKE_CXX_FLAGS "${LEATHERMAN_CXX_FLAGS} ${CPP_PCP_CLIENT_FLAGS}")
 add_definitions(${LEATHERMAN_DEFINITIONS})
 
+# Disable SSLv2 and SSLv3 (in Boost.Asio) for reals.
+add_definitions(-DOPENSSL_NO_SSL2 -DOPENSSL_NO_SSL3)
```

Comment:
This seems to be magically set somehow on my Mac. Haven't figured out where yet.